distribution/changelog.txt

```diff
@@ -16,6 +16,7 @@
 - Fix: [#14649] ImageImporter incorrectly remaps colours outside the RCT2 palette.
 - Fix: [#14667] “Extreme Hawaiian Island” has unpurchaseable land tiles (original bug).
 - Fix: [#15096] Crash when placing entrances in the scenario editor near the map corner.
+- Fix: [#15136] SV6 export causing vanilla RCT2 to crash under certain situations.
```

Comment:
It was a freeze and not a crash. Also, I think it was in most or even all situations.

```suggestion
- Fix: [#15136] Exported SV6 files cause vanilla RCT2 to hang.
```

Reply:
Hmm well that means i fixed two bugs then as it was crashing this morning after only the e16a4a0 commit.